integration-tests/cli/csv-parser-price.it.js

```diff
@@ -55,12 +55,13 @@ describe('CSV and CLI Tests', () => {
       })
     })
 
-    test('should take input from file', (done) => {
+    // FIXME: fails with `ERR! No type with key \'custom-type\' found`
```

Comment:
@hisabimbola @wizzy25 FYI: for some reason this was failing. It would be nice if you guys can have a look when you have time. Thanks!